geoh5py/workspace/workspace.py

```diff
@@ -225,6 +225,7 @@ def close(self):
                     check=True,
                     shell=True,
```

Comment:
preferably, avoid using `Shell=True`. See [subprocess.html#security-consideration](https://docs.python.org/3/library/subprocess.html#security-considerations)).
Code be changed to:
```py
                subprocess.run(
                    ["h5repack", "--native", str(self._h5file), str(temp_file)],
                    check=True,
```
(`shell=False` is the default)